packages/grafana-ui/src/components/DataLinks/FieldLinkList.tsx

```diff
@@ -30,7 +31,9 @@ export function FieldLinkList({ links }: Props) {
         return <DataLinkButton key={i} link={link} />;
       })}
       <div className={styles.wrapper}>
-        <p className={styles.externalLinksHeading}>External links</p>
+        <p className={styles.externalLinksHeading}>
+          <Trans i18nKey="grafana-ui.field-link-list.external-links-heading">External links</Trans>
```

Comment:
Should a paragraph be named as a heading?

Reply:
i think if anything it's _more_ correct to call it a heading, and the actual problem is using a `<p>` element for (essentially) spacing reasons. but i'm definitely not fixing that as part of this PR 😅